readme.md

```diff
@@ -81,18 +83,25 @@ Your Rails database has two tables. `students` has the columns `id` and `name`,
 Use ActiveRecord to create a new `pbj` sandwich and make it belong to the student named Geraldo.
 
 ```rb
-# Your answer...
+geradlo = Student.create(name: "Geraldo")
+Sandwich.create(type: "pbj", student_id: "Geraldo")
```

Comment:
Almost, but not quite...
the last line needs to reference the ID of the geraldo student/object:

```ruby
Sandwich.create(type: "pbj", student_id: geraldo.id)
# or 
Sandwich.create(type: "pbj", student: geraldo)
```